src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -31,6 +31,9 @@ interface VcsRootLocator {
 }
 
 interface BuildLocator {
+    fun withId(buildId: BuildId): BuildLocator
```

Comment:
It looks like we have different opinions about goals of this library. It isn't supposed to be be dummy wrapper around REST API for those who know REST API. I think it's much better to provide convenient access to TeamCity for those who just want to interact with TeamCity in scripts. So it shouldn't keep names of attributes in REST API if we can invent more comprehensible names. If REST provides too generic API for something and we don't know about real use cases for that it's better to provide more specific API in the library to make it easier to use.

I already had similar discussion in other pull requests (e.g. [here](https://github.com/JetBrains/teamcity-rest-client/pull/15#discussion_r120849639)). If you don't agree with me let's discuss this in person.